### PR TITLE
Add new links to Resources.

### PR DESCRIPTION
**Pull request by brev**
Add new links to Resources.

### CODE REVIEW

**Review by GeoffreyBooth — APPROVED** [review]

File: doc/resources.md [hunk 1]
```diff
@@ -18,6 +18,8 @@
 
 * [babel-register-esm](https://github.com/giltayar/babel-register-esm)
 
+* [create-esm-loader](https://github.com/sebamarynissen/create-esm-loader#readme) and [node-esm-loader](https://github.com/sebamarynissen/node-esm-loader#readme), along with [several compatible loaders](https://github.com/brev/esm-loaders#readme) (typescript, svelte, css, mocking, etc)
```
Comment (JakobJingleheimer) [inline doc/resources.md:21]: Are these related/dependent to each other? I think no (the first describes itself as "_a utility library for creating esm loader hooks_", the second describes itself as "_webpack-like loader library for Node_"). If not, I think these should be separate list items.

Comment (brev) [reply doc/resources.md:21]: Hi @JakobJingleheimer, yes, they are intertwined. The first helps create each loader. The second helps load one/many of those previously created loaders. Thanks.

**Review by JakobJingleheimer — APPROVED** [review]
